src/mudlet_manual.txt

```diff
@@ -867,8 +867,6 @@ to parse output that is known to arrive in a certain line margin or to delete un
 tempLineTrigger( 20, 2, [[send("shout Help")]] )
```

Comment:
Just a side note `src/mudlet_manual.txt` needs to go, we've migrated everyone to the wiki years ago and that file is outdated and unused.